activesupport/lib/active_support/cache.rb

```diff
@@ -464,19 +464,26 @@ def fetch_multi(*names)
         options = names.extract_options!
         options = merged_options(options)
 
-        instrument :read_multi, names, options do |payload|
-          reads   = read_multi_entries(names, **options)
-          writes  = {}
-          ordered = names.index_with do |name|
-            reads.fetch(name) { writes[name] = yield(name) }
-          end
+        if options[:force]
```

Comment:
Can you document this in the above comments (same as for `#fetch`)

Reply:
Then once that's done could you please squash your commits and add a changelog entry.